File: backend/Models/Conference.cs

public class Conference{
	public Guid Id{ get; set;}
	
	public required string Name{ get; set;}

	public DateTime Start {get; set;}

	public DateTime End{get; set;}

	public string? Description{get; set;}

	public ICollection<UploadedFile> Files { get; } = new List<UploadedFile>();
}

File: src/app/models/conference.ts
import { UploadedFile } from "./uploadedFile"

export class Conference{
	id: string
	name: string
	start: Date
	end: Date
	description?: string
	files?: UploadedFile[]

	constructor(details: Conference) {
		this.id = details.id
		this.name = details.name
		this.start = details.start
		this.end = details.end
		this.description = details.description
		this.files = details.files
	}
}
